boot/boot.js

```diff
@@ -1444,6 +1444,7 @@ $tw.loadTiddlersBrowser = function() {
 	for(var t=0; t<containerIds.length; t++) {
 		$tw.wiki.addTiddlers($tw.wiki.deserializeTiddlers("(DOM)",document.getElementById(containerIds[t])));
 	}
+	callback();
```

Comment:
This is a rough approximation of `$tw.boot.startup`.

```js
function startup(){
    $tw.loadSettings();
    $tw.loadEnvironment();
    $tw.loadTiddlersNode();
    $tw.loadPlugins();
}
```

Then in my own code I write. 

```js
function init(){
    $tw = require('tiddlywiki').TiddlyWiki();
    makeLoadAsync($tw);
    $tw.boot.boot();
}
function makeLoadAsync($tw){
    $tw.loadTiddlersNode = function(){
        //async'ly load a whole bunch of files instead of 
        //sync'ly so that the event loop can do other stuff 
        //in my program while Node is reading files from the 
        //file system.
    }
}
```

Obviously that won't work since there will be no plugins available. So instead I want to do this.

```js
function startup(){
    $tw.loadSettings();
    $tw.loadEnvironment();
    $tw.loadTiddlersNode(() => {
        $tw.loadPlugins();
    });
}
```

But I don't care if loadTiddlersNode returns sync'ly, because I'm not doing anything else until it returns anyway. This is only to allow others to write code that does not block the event loop if they want to.